camelot/core/files/storage.py

```diff
@@ -224,12 +228,22 @@ def delete(self, name: PurePath):
     def _process_path(self, path: PurePath) -> PurePath:
         return PurePath(os.path.relpath(path, start=self.upload_to))
 
+    def _verbose_name(self, path: PurePath, name_hint: str) -> str:
```

Comment:
ik zou _verbose_name als volgt implementeren : 

- is er een name_hint, dan is de verbose name gewoon de name hint, dan mag specialisatie in HashStorage weg
- is er geen name_hint, dan is verbose name de laatste component vd path (dus nt de volledige path)

de verbose name is een 'nuttige weergave voor de gebruiker', daar hoeft dus voor de rest nt te veel in te staan ...

Reply:
Ja het probleem is dat wanneer we Hashed files van db of disk halen is er geen nuttige weergave voor de gebruiker en er is ook geen manier om een nuttige weergave te maken. Dus we kunnen idd beter doen zoals je zegt

Reply:
klopt, dat is dan ook een volgend probleem ;)